scene/3d/xr_body_modifier_3d.h

```diff
@@ -77,20 +74,21 @@ class XRBodyModifier3D : public Node3D {
 protected:
 	static void _bind_methods();
 
+	virtual void _skeleton_changed(Skeleton3D *p_old, Skeleton3D *p_new) override;
+	virtual void _process_modification() override;
+
 private:
 	struct JointData {
 		int bone = -1;
 		int parent_joint = -1;
 	};
 
 	StringName tracker_name = "/user/body";
-	NodePath target;
 	BitField<BodyUpdate> body_update = BODY_UPDATE_UPPER_BODY | BODY_UPDATE_LOWER_BODY | BODY_UPDATE_HANDS;
 	BoneUpdate bone_update = BONE_UPDATE_FULL;
 	bool show_when_tracked = true;
 	JointData joints[XRBodyTracker::JOINT_MAX];
 
-	Skeleton3D *get_skeleton();
 	void _get_joint_data();
 	void _update_skeleton();
```

Comment:
`void XRBodyModifier3D::_update_skeleton()` shadows non-virtual method `void SkeletonModifier3D::_update_skeleton()`

These methods perform different actions: one updates the skeleton reference in SkeletonModifier3D, while the other assigns bone rotations during processing.
We should rename one of them

Reply:
Oh my god! Renamed!